pkg/pool-weighted/test/WeightedPoolFactory.test.ts

```diff
@@ -26,17 +31,29 @@ describe('WeightedPoolFactory', function () {
   let createTime: BigNumber;
 
   sharedBeforeEach('deploy factory & tokens', async () => {
-    const vault = await Vault.create();
+    vault = await Vault.create();
 
     factory = await deploy('WeightedPoolFactory', { args: [vault.address] });
     createTime = await currentTimestamp();
 
     tokens = await TokenList.create(['MKR', 'DAI', 'SNX', 'BAT'], { sorted: true });
+
+    [, assetManager] = await ethers.getSigners();
```

Comment:
Can you move this into 
```
  before('setup signers', async () => {
    [, assetManager] = await ethers.getSigners();
  });
```

Just so that it looks like every other test file. Thanks!